src/content/app/entity-viewer/gene-view/components/protein-domain-image/ProteinDomainImage.tsx

```diff
@@ -35,12 +35,17 @@ export type ProteinDomainImageProps = {
   };
 };
 
+type location = {
```

Comment:
- we begin type names with a capital letter
- however, we already have a `Location` type elsewhere

Reply:
I cant see where we have `Location` type, we do have a `location.ts` and a type `LocationWithinRegion` but not just `Location`. i know we used it elsewhere but i dont think its exported anywhere else.